### PR TITLE
Encription

### PR DESCRIPTION
**Pull request by faisalakb**
In this project apply the following features: :100: 

- Create a method to decode a Morse code character, takes a string parameter, and return the corresponding character in  uppercase (e.g. decode_char(".-") returns "A"). :+1: 

- Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY"). :+1: 

- Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces. :+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +123,7 @@ If you like this project, please consider giving it a star.
 
 ## 📝 License <a name="license"></a>
 
+
 This project is [MIT](https://github.com/faisalakb/Ruby_Morse_Code.git/blob/dev/LICENSE) licensed.
```
Comment (1GORDON) [inline README.md]: Great job team 🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.
MiT license

File: README.md [hunk 2]
```diff
@@ -71,7 +72,7 @@ git clone https://github.com/faisalakb/Ruby_Morse_Code.git
 Install this project with:
 
 ```
-cd cd Create-your-own-Enumerable
+cd Ruby_Morse_Code
 gem install
 ```
```
Comment (1GORDON) [inline README.md:78]: Great job team 🤝

- [ ] Kindly fix the broken issues link to make your readme file more professional.
Issues page

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @faisalakb, and @wacoo ✋

Good job so far!

## Highlights!
- Project has `#decode_char` method ✔️ 
- Project has `#decode_word` method ✔️
- Project has `#decode` method ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Alusp) [reply README.md:78]: - [ ] Good job in implementing the changes requested by the previous reviewer, :clap:  however as he mentioned above please ensure that you solve your broken issues link. Having broken links in your documentation can make your project or organization appear unprofessional and unreliable. Users may question the accuracy and trustworthiness of your content. :slightly_smiling_face: 

Below is the screenshot: :arrow_down: 
Issu

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @faisalakb, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
close

 ## Highlights
✔️ Project has #decode_char method ✔️
✔️ Project has #decode_word method ✔️
✔️ Project has #decode method ✔️
 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by faisalakb** [comment]
@Alusp please, could you check it again. Because when you looked into the feature branch(encription branch) every link is working fine. Why you are checking in Dev branch ? I will be merge it after approval. could you please attach the small video chunk for my understanding where the link is broken; when you click on LICENSE Link.

**Comment by Alusp** [comment]
hello @faisalakb, I believe you misunderstood my post, I said the link to your `issues` section, not your `License`. here is a video demonstration: [here](https://www.loom.com/share/55164e796c4b45fd8731417ffcc375e6?sid=32ae53ed-dfff-4cc3-b622-1d6cd75a377c)

**Comment by faisalakb** [comment]
@Alusp my apologies yes you are right . Now I fix the broken issues link.

**Review by Efiamotu-1 — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

image

## Highlights :dart: 
* Correct GitHub flow or Gitflow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**